teemaderegister-react/src/constants/ActionTypes.js

```diff
@@ -55,3 +55,5 @@ export const USER_PICTURE_SET_END = 'USER_PICTURE_SET_END'
 
 export const PASSWORD_CHANGE_START = 'PASSWORD_CHANGE START'
 export const PASSWORD_CHANGE_END = 'PASSWORD_CHANGE_END'
+
+export const ADMIN_INIT_ADD_NEW_USER = 'INIT_ADD_NEW_USER'
```

Comment:
I think this is not used anymore, or is it?

Reply:
No it's not, it's redundant